app/Http/Controllers/Auth/LoginController.php

```diff
@@ -36,4 +37,21 @@ public function __construct()
     {
         $this->middleware('guest')->except('logout');
     }
+
+    /**
+     * Log the user out of the application.
+     *
+     * @param  \Illuminate\Http\Request  $request
```

Comment:
Personally I like to simplify classnames in the code by using them at the top of the file (as you did with this Request class). You can now simply refer to this class with 'Request' (no namespace needed).